src/main/java/com/client/core/base/tools/entitychanger/impl/StandardEntityChanger.java

```diff
@@ -103,6 +103,8 @@ public <T> T setField(T entity, String field, Object value) {
                         propertyDescriptor.getWriteMethod().invoke(target, asType(value, BigDecimal.class));
                     } else if (Boolean.class.equals(fieldType)) {
                         propertyDescriptor.getWriteMethod().invoke(target, asType(value, Boolean.class));
+                    } else if (String.class.equals(fieldType)) {
+                        propertyDescriptor.getWriteMethod().invoke(target, asType(value, String.class));
```

Comment:
Why can't we generalize these lines to:

```suggestion
                        propertyDescriptor.getWriteMethod().invoke(target, asType(value, fieldType));
```

Reply:
Applied this suggestion to the whole if-statement